bacteria/access.md

```diff
@@ -1,4 +1,4 @@
 - *Search for a gene* - type the name of a gene or other identifier into the search box above
 - *Find a genome* - click in the 'browse a genome' box above and start typing your genome name to find matching genomes
 - [View full list of all Ensembl Bacteria species](/species.html)
-- [Access Ensembl Bacteria programmatically](/info/data/api.html)
+- [Access Ensembl Bacteria programmatically] (/info/data/api.html)

```

Comment:
Sorry, one more thing. This space will break the link. Markdown syntax requires no space between the text of the link in square brackets and the url of the link in parentheses.

Reply:
Oops sorry. Done.